app/controllers/resources_controller.rb

```diff
@@ -57,6 +57,15 @@ def destroy
     end
   end
 
+  # Return the total number of active (i.e., approved) resources
+  def count
+    if !admin_signed_in?
```

Comment:
This functionality is needed for the new landing page, so shouldn't require sign-in.

Reply:
Great, will fixup.